tss/ws/server/handler.go

```diff
@@ -160,19 +192,29 @@ func (wm *WebsocketManager) WebsocketHandler(w http.ResponseWriter, r *http.Requ
 	}()
 
 	pubKey := r.Header.Get("pubKey")
-	timeStr := r.Header.Get("time")
+	if len(pubKey) < publicKeyLength {
+		wm.logger.Error("Failed to establish connection", "err", errors.New("invalid pubKey in header"))
```

Comment:
Please print detailed error message, for instance: fmt.Errorf("invalid pubKey in header, expected length %d, actual length", publicKeyLength, len(pubKey))